build/template-install-dependencies.yaml

```diff
@@ -3,6 +3,26 @@
 #install dotnet core
 
 steps:

```

Comment:
If this is actually what we need to do, lets do it as a strategy matrix and test per target.